CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ nav_order: 1
 
 # Changelog
 
+## [MAJOR.MINOR.PATCH] - YYYY-MM-DD
+
+- Add additional disc space support
```

Comment:
please change all `disc` to `disk` everywhere, see: https://en.wikipedia.org/wiki/Spelling_of_disc

Reply:
done :+1: